roles/ansible/tower/manage-workflow-templates/tasks/process-workflow-template.yml

```diff
@@ -0,0 +1,84 @@
+---
+
+- name: "Remove the workflow template if currently defined"
```

Comment:
This means we are *always* removing and re-adding workflows that exists - even the ones without any changes. We should look for a better solution for this. I have not had a chance to look, but I presume the API supports updating an existing workflow, so let's look to use that in the case where a workflow already exists.